File: jsonrpc4j-ep/src/main/java/jsonrpc4jep/jsonrpc4jep/Person.java
package jsonrpc4jep.jsonrpc4jep;

public class Person {

    String nome;
    int id;
    String email;

    enum TipoTel {MOBILE, HOME, WORK};

    private class PhoneNum {
        public String number;
        public TipoTel tipo;
    }

    public Person() {

    }


}

File: json-rpc/Person.py
from enum import Enum

class PhoneType(Enum):
    MOBILE = 0
    HOME = 1
    WORK = 2

class Status(Enum):
    CADASTRADO = 0
    NAOCADASTRADO = 1

class PhoneNumber:
    number = ''
    typePhone = ''

class MyPerson:
    name = ''
    email = ''
    phones = [PhoneNumber]

class PersonReply:
    id = 0
    name = ''
    stat = 0

